consolidate/mesher/meshes.py

```diff
@@ -6,48 +6,88 @@ class MeshTwoPlates():
     def __init__(self, deck):
         self.deck = deck
         self.set_mesh_grid() 
-        self.init_mesh() 
      
     def set_mesh_grid(self):
-        self.nx = int(float(self.deck.doc["Simulation"]["lenX"])/float(self.deck.doc["Simulation"]["dx"]))
-        self.ny = int(float(self.deck.doc["Simulation"]["lenY"])/float(self.deck.doc["Simulation"]["dy"]))
+        self.nx = int(self.deck.doc["Simulation"]["Number of Elements X"])
+        self.ny = int(self.deck.doc["Simulation"]["Number of Elements Y"])
+        self.q=float(self.deck.doc["Processing Parameters"]["Power Density"])
+        self.Lx=float(self.deck.doc["Simulation"]["lenX"])
+        self.Ly=float(self.deck.doc["Simulation"]["lenY"])
+        self.dx=self.Lx/self.nx
+        self.dy=self.Ly/self.ny
         X, Y = np.meshgrid(np.arange(0, self.ny), np.arange(0, self.nx))
         X=X[1,:].copy()
         Y=Y[:,1].copy()
         self.X = X
         self.Y = Y
 
-    def init_mesh(self):
-
         self.nx1, self.nx2 = self.nx, self.nx
         self.ny1, self.ny2 = int(self.ny/2), self.ny
-        
+
+  
+
+# -------------- BEGIN DEFINING TEMPERATURE INITIAL CONDITION ----------
         T = np.zeros((self.ny, self.nx))        
         T[0:self.ny1, 0:self.nx1] = self.deck.doc["Materials"]["Material1"]["Domain Initial Temperature"] # Set array size and set the interior value with Tini
         T[self.ny1:self.ny2, 0:self.nx2] = self.deck.doc["Materials"]["Material2"]["Domain Initial Temperature"] # Set array size and set the interior value with Tini
-        T[int(self.ny/2), 0:] = self.deck.doc["Processing Parameters"]["Temperature"]
-        T[int(self.ny/2-1), 0:] = self.deck.doc["Processing Parameters"]["Temperature"]
+        T[int(self.ny/2), 1:-1] = self.deck.doc["Processing Parameters"]["Temperature"]
+        T[int(self.ny/2-1), 1:-1] = self.deck.doc["Processing Parameters"]["Temperature"]
         self.T = T.copy()
         self.T0=T.copy()
+# -------------- END DEFINING TEMPERATURE INITIAL CONDITION ----------
+             
+# ------------ BEGIN THERMAL CONDUCTIVITY INITIAL CONDITION ----------        
```

Comment:
These could be made into functions of the class to make the code easier to read.

Reply:
solved in https://github.com/ets-lipec/consolidation-works/pull/5/commits/8255b6eb3e146b4fe14164a5b489e21c5c6d634e